src/pages/home/base/mobile-core/api-reference.md

```diff
@@ -169,20 +169,31 @@ This API initializes AEP SDKs by automatically registering all extensions bundle
 
 Two versions of this API are available, which accept **app ID** or **InitOptions**:
 
-- __app ID__: Configures the SDK with the provided mobile property environment ID configured from the Data Collection UI.
-- __InitOptions__: Allow customization of the default initialization behavior. Refer [InitOptions](#initoptions).
+* **app ID**: Configures the SDK with the provided mobile property environment ID configured from the Data Collection UI.
 
-This API eliminates the need to register extensions manually using [registerExtensions](#registerextensions) and manage lifecycle tracking with [lifecycleStart](lifecycle/api-reference.md#lifecyclestart) and [lifecyclePause](lifecycle/api-reference.md#lifecyclepause) APIs. 
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
+
+Android
+
+<Tabs query="platform=android&api=initialize-appid"/>
+
+iOS
+
+<Tabs query="platform=ios&api=initialize-appid"/>
+
+* **InitOptions**: Allow customization of the default initialization behavior. Refer [InitOptions](#initoptions).
 
 <TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
 
 Android
 
-<Tabs query="platform=android&api=initialize"/>
+<Tabs query="platform=android&api=initialize-initoptions"/>
 
 iOS
 
-<Tabs query="platform=ios&api=initialize"/>
+<Tabs query="platform=ios&api=initialize-initoptions"/>
+
+This API eliminates the need to register extensions manually using [registerExtensions](#registerextensions) and manage lifecycle tracking with [lifecycleStart](lifecycle/api-reference.md#lifecyclestart) and [lifecyclePause](lifecycle/api-reference.md#lifecyclepause) APIs.
```

Comment:
Consider using a `[!NOTE]` or even a `[!IMPORTANT]` here to highlight that these APIs replace registerExtensions and Lifecycle APIs.

Reply:
I think an IMPORTANT would be good here, since it's kind of important. Maybe have that earlier on?

Reply:
Do those work with our docs or do you mean `InlineAlert`?

Reply:
Yes, I'm sorry. I was thinking about our GitHub docs. I mean use an InlineAlert.